
module.exports.err = {
  ERR_DURING_DELETE_RECORD: 'error during deleteing record',
}